lib/shared/comment_header.dart

```diff
@@ -71,85 +76,100 @@ class CommentHeader extends StatelessWidget {
+                                                    size: 15.0 * state.metadataFontSizeScale.textScaleFactor,
+                                                    color: theme.colorScheme.onBackground,
+                                                  ))
+                                              : Container(),
+                                        ),
+                                        Container(
+                                          child: isAdmin(comment.creator)
+                                              ? Padding(
+                                                  padding: const EdgeInsets.only(left: 1),
+                                                  child: Icon(
+                                                    Thunder.shield_crown,
+                                                    size: 14.0 * state.metadataFontSizeScale.textScaleFactor,
+                                                    color: theme.colorScheme.onBackground,
+                                                  ),
+                                                )
+                                              : Container(),
+                                        ),
+                                        Container(
+                                          child: isModerator(comment.creator, moderators)
+                                              ? Padding(
+                                                  padding: const EdgeInsets.only(left: 1),
+                                                  child: Icon(
+                                                    Thunder.shield,
+                                                    size: 14.0 * state.metadataFontSizeScale.textScaleFactor,
+                                                    color: theme.colorScheme.onBackground,
+                                                  ),
+                                                )
+                                              : Container(),
+                                        ),
+                                        Container(
+                                          child: isBot(comment.creator)
+                                              ? Padding(
+                                                  padding: const EdgeInsets.only(left: 1, right: 2),
+                                                  child: Icon(
+                                                    Thunder.robot,
+                                                    size: 13.0 * state.metadataFontSizeScale.textScaleFactor,
+                                                    color: theme.colorScheme.onBackground,
+                                                  ),
+                                                )
+                                              : Container(),
+                                        ),
+                                      ],
+                                    )
+                                  : RichText(
```

Comment:
Does this respect the font scaling applied by the user? (`state.metadataFontSizeScale`). I believe you can follow something like this to get the scaling to work here:

- Set the font size to `MediaQuery.textScalerOf(context).scale(theme.textTheme.bodyMedium!.fontSize! * state.metadataFontSizeScale.textScaleFactor)`
- Set the text scaler on the `RichText` to be `textScaler: TextScaler.noScaling`

This was taken from `post_card_view_compact.dart` if you're curious!
```dart
Text.rich(
  TextSpan(
    children: [
      TextSpan(
        text: HtmlUnescape().convert(postViewMedia.postView.post.name),
        style: theme.textTheme.bodyMedium?.copyWith(
          fontWeight: FontWeight.w600,
          fontSize: MediaQuery.textScalerOf(context).scale(theme.textTheme.bodyMedium!.fontSize! * state.titleFontSizeScale.textScaleFactor),
          color: postViewMedia.postView.post.featuredCommunity
              ? (indicateRead && postViewMedia.postView.read ? Colors.green.withOpacity(0.55) : Colors.green)
              : (indicateRead && postViewMedia.postView.read ? theme.textTheme.bodyMedium?.color?.withOpacity(0.55) : null),
        ),
      ),
    ],
  ),
  textScaler: TextScaler.noScaling,
),
```